lib/Form.tsx

```diff
@@ -84,14 +84,30 @@ export default function Form({
         throw err;
       }
 
-      err.inner.forEach((error: ValidationError) => {
+      err.inner.forEach((error: Yup.ValidationError) => {
         validationErrors[error.path] = error.message;
       });
 
       setErrors(validationErrors);
     }
   }
 
+  async function onBlurValidation(
```

Comment:
I think that we should not use `event` here, maybe we should get the input value in the same way that we get in `parseFormData` because some form elements may not have this event with `name` and `value` properties. 

Maybe we should also create an function called getFieldValue and use the same function here and inside `parseFormData`